pkg/util/paramtable/service_param.go

```diff
@@ -675,10 +675,16 @@ func (p *PulsarConfig) Init(base *BaseTable) {
 // --- kafka ---
 type KafkaConfig struct {
 	Address             ParamItem  `refreshable:"false"`
+	SaslEnabled         ParamItem  `refreshable:"false"`
 	SaslUsername        ParamItem  `refreshable:"false"`
 	SaslPassword        ParamItem  `refreshable:"false"`
 	SaslMechanisms      ParamItem  `refreshable:"false"`
-	SecurityProtocol    ParamItem  `refreshable:"false"`
+	SslEnabled          ParamItem  `refreshable:"false"`
```

Comment:
Kafka SSL has been supported recently by this pr: https://github.com/milvus-io/milvus/pull/30468, and this pr has been tested with Kafka in the 4 ways:
1. no SASL, no SSL
2. SASL only
3. SSL only
4. SASL+SSL

Your code change conflicts with this pr. Please remove the code change about kafka ssl, keep the pr only for minio ssl.